operator/Makefile

```diff
@@ -65,6 +65,12 @@ INDEX_IMG ?= $(INDEX_IMG_BASE):v$(VERSION)
 # Image URL to use all building/pushing image targets
 IMG ?= $(IMAGE_TAG_BASE):$(VERSION)
 
+# If OPERATOR_IMAGE_DIGEST is set, overwrite the operator image to use using this digest.
+# Used for image pinning.
+ifneq ($(OPERATOR_IMAGE_DIGEST),)
+IMG = "$(IMAGE_TAG_BASE)@$(OPERATOR_IMAGE_DIGEST)"
+endif
```

Comment:
It feels wrong to override `IMG` for the purpose of pinning the digest.

For instance, there's `docker-build` target that uses `IMG` and feeding it digest is confusing and likely results in error. I know we don't use `docker-build` in Konflux but it still feels wrong to complicate understanding of this Makefile by forcing the reader to know contexts.

Since pinning is expected to happen close-to or directly in `patch-csv.py`, could you please handle the digest there?

Reply:
Probably after `patch-csv.py` and potentially as a separate executable. The reason is that downstream (which we shouldn't break yet) uses `patch-csv.py` but does pinning on its own and using its own logic.